src/test/groovy/edu/ie3/datamodel/io/source/csv/CsvTimeSeriesSourceIT.groovy

```diff
@@ -67,7 +66,7 @@ class CsvTimeSeriesSourceIT extends Specification implements CsvTestDataMeta {
 
 	def "A csv time series source is able to return a time series for a period of interest"() {
 		given:
-		def interval = new ClosedInterval(TimeUtil.withDefaults.toZonedDateTime("2020-01-01 00:15:00"), TimeUtil.withDefaults.toZonedDateTime("2020-01-01 00:15:00"))
+		def interval = new ClosedInterval(TIME_15MIN, TIME_15MIN)
```

Comment:
```suggestion
		def interval = new ClosedInterval(TIME_00MIN, TIME_15MIN)
```

Reply:
That would change the original test. Maybe you can explain why this is better?